raw_linux.go

```diff
@@ -0,0 +1,242 @@
+// +build go1.12
```

Comment:
These still needed? `go.mod` already has go1.13 as indicator for "minimum supported version"; https://github.com/containerd/fifo/blob/fcd6de7ee90bf3789140ed2f508be259ce8f48ff/go.mod#L3

Reply:
+1 for removing the line.

Reply:
Removed.